doc/source/reliability/show_slave_status_nolock.rst

```diff
@@ -20,7 +20,7 @@ This will display the slave's status as if there were no lock, allowing the user
 
 .. note::
 
-   |Percona Server| originally used ``SHOW SLAVE STATUS NOLOCK`` syntax for this feature. As of :rn:`5.6.20-68.0` release, |Percona Server| implements ``SHOW SLAVE STATUS NONBLOCKING`` syntax, which comes from |MySQL| 5.7. The ``NOLOCK`` one has been deprecated and will be removed in |Percona Server| 5.7.
+   |Percona Server| originally used ``SHOW SLAVE STATUS NOLOCK`` syntax for this feature. As of :rn:`5.6.20-68.0` release, |Percona Server| implements ``SHOW SLAVE STATUS NONBLOCKING`` syntax, which comes from |MySQL| 5.7. 
```

Comment:
which comes from early MySQL 5.7. Note that current MySQL 5.7 does not have this syntax and regular "SHOW SLAVE STATUS" is non-blocking.